web/js/app.js

```diff
@@ -374,14 +498,15 @@ LR.utils.broadcastEvent = function(event = {}, type, element, obj = {}) {
  * @param {string} The @#id of the experience to disaasociate it from
  * @return {Promise}
  */
-LR.utils.disassociateObject = function(event, objectID, experienceID){
-    let trackedObjs = document.getElementById("objects").value
-    let delim = document.getElementById("objects").hasAttribute("deer-array-delimeter") ? document.getElementById("objects").getAttribute("deer-array-delimeter") : ","
+LR.utils.disassociate = function(event, objectID, experienceID, which){
+    let inpt = document.querySelector("input[deer-key='"+which+"']")
```

Comment:
An aside <kbd>PROTIP</kbd>: you can use template literals in spaces like this and it makes it easier to read. I always get lost in my `'"+var+"'` patterns and you can instead take `tag[attr='val']` selectors and make them flexible like `tag[attr='${}val']`. This is also great when you start using `*=` and `^=` selectors, sicne you can put regex stuff inside of `${}`.